README.md

```diff
@@ -65,6 +65,9 @@ Enables you to create more customized BLE Peripheral's compared to the basic UAR
    * LC
  * [MPIDE](http://chipkit.net/started/install-chipkit-software/) ([ChipKit](http://chipkit.net))
    * PIC32MX
+ * [Arduino](https://www.arduino.cc/en/Main/arduinoBoardMega) ([Arduino Mega](https://www.arduino.cc/en/Main/arduinoBoardMega))
+   * Arduino MEGA
+   * You may experience memory problems on Arduino boards with 32kb flash storage (e.g. Uno,Leonardo)
```

Comment:
What would be correct way to put warning for gullible UNO users like me? :D 
So they can order a proper board, before starting the project. I just switched to Teensy 3.2 and everything is running perfect. This library definitely worths the space it takes. It is very advanced compared to original SDK for nrf8001.

But I believe it is good to put a warning, to avoid causing disappointment.

Because Arduino UNO's memory does not seem enough to do much. If you don't have any extra code, it is able to advertise bluetooth. If you run BT_HID test which aims to run all HID devices, then it is not able to pass setup stage.